cves/CVE-2018-13304.yml

```diff
@@ -209,8 +215,10 @@ specification:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: While these fixes where leading to a specification for
+    the codec profile, it is clear that this specification did not exist
```

Comment:
Did FFmpeg create this specification then? I'm not sure if I follow, as most of these new features and codecs are based on industry specficiations.